src/app.js

```diff
@@ -1 +1,34 @@
 'use strict';
+
+const fs = require('fs');
+
+const path = require('path');
+
+const copyFile = (src, dest) => {
+  const srcPath = path.join(__dirname, src);
+  const destPath = path.join(__dirname, dest);
+
+  if (srcPath === destPath) {
+    return;
```

Comment:
You should throw an error or print a message when source and destination are the same